frontend/src/metabase/containers/MoveModal.tsx

```diff
@@ -68,7 +68,7 @@ export const MoveModal = ({
         id: initialCollectionId,
         model: "collection",
       }}
-      onChange={async newCollection => await onMove({ id: newCollection.id })}
+      onChange={newCollection => onMove({ id: newCollection.id })}
```

Comment:
i think errors in the move modal will not propagate properly if we remove this `await`

#40700